heidrun/mdl_map.rb

```diff
@@ -0,0 +1,77 @@
+Krikri::Mapper.define(:mdl_map, :parser => Krikri::JsonParser) do
```

Comment:
I think you should be able to do this by passing `'record'` as a third argument. i.e.  `Krikri::Mapper.define(:mdl_map, :parser => Krikri::JsonParser, 'record')`

The relevant code is:

  - https://github.com/dpla/KriKri/blob/develop/lib/krikri/mapping.rb#L22
  - https://github.com/dpla/KriKri/blob/develop/lib/krikri/mapping.rb#L35
  - https://github.com/dpla/KriKri/blob/develop/lib/krikri/parsers/json_parser.rb#L14

Extra arguments are forwarded to the parser. I'm not 100% certain that `'record'` is the correct `root_path`, but that seems right.